inst/Upgrades.qml

```diff
@@ -0,0 +1,289 @@
+		{
+			name:		"testValues"
+			jsFunction:	function(options)
+			{
+				switch(options["testValues"])
+				{
+					case "multinomialTest":		return "equal";
+					case "expectedProbs":		return "custom";
+				}
+			}
+		}
+		ChangeRename{	from: "tableWidget";					to: "testValuesCustom"			}
+		ChangeRename{	from: "descriptives";					to: "descriptiveTable"			}
+		ChangeRename{	from: "credibleInterval";				to: "descriptiveTableCi"		}
+		ChangeRename{	from: "credibleIntervalInterval";		to: "descriptiveTableCiLevel"	}
+		ChangeRename{	from: "countProp";						to: "descriptivesAs"			}
+		ChangeJS
+		{
+			name: "descriptivesAs"
+			jsFunction:	function(options)
+			{
+				switch(options["descriptivesAs"])
+				{
+					case "descCounts":		return "counts";
+					case "descProps":		return "proportions";
+				}
+			}
+		}
+		ChangeRename{	from: "descriptivesPlot";						to: "descriptivePlot"			}
+		ChangeRename{	from: "descriptivesPlotCredibleInterval";		to: "descriptivePlotCiLevel"	}
+	}
+
+	Upgrade
+	{
+		functionName:		"ContingencyTables"
+		fromVersion:		"0.15"
+		toVersion:			"0.16.4"
+
+		ChangeRename{	from: "VovkSellkeMPR";							to: "vovkSellke"				}
+		ChangeRename{	from: "LogOdds";								to: "oddsRatioAsLogOdds"		}
+		ChangeRename{	from: "oddsRatioConfidenceIntervalInterval";	to: "oddsRatioCiLevel"			}
+		ChangeRename{	from: "oddsRatioHypothesis";					to: "oddsRatioAlternative"		}
+		ChangeSetValue
+		{
+			name:		"oddsRatioAlternative"
+			condition:	function(options) { return options["oddsRatioAlternative"] === "two.sided" }
+			jsonValue:	"twoSided"
+		}
+		ChangeRename{	from: "cochransAndMantel";						to: "cochranAndMantel"						}
+		ChangeRename{	from: "testOddsRatioEquals";					to: "cochranAndMantelCommonOddsRatioTest"	}
+		ChangeRename{	from: "hideSmallCounts";						to: "countsHiddenSmallCounts"				}
+		ChangeRename{	from: "hideSmallCountsLessThan";				to: "countsHiddenSmallCountsThreshold"		}
+		ChangeRename{	from: "zTestCompareColumns";					to: "zTestColumnComparison"					}
+		ChangeRename{	from: "zTestAdjustPValues";						to: "zTestAdjustedPValues"					}
+	}
+
+	Upgrade
+	{
+		functionName:		"ContingencyTablesBayesian"
+		fromVersion:		"0.15"
+		toVersion:			"0.16.4"
+
+		ChangeRename{	from: "oddsRatioCredibleIntervalInterval";			to: "oddsRatioCiLevel"	}
+		ChangeRename{	from: "effectSize";									to: "cramersV"	}
+		ChangeRename{	from: "effectSizeCredibleIntervalInterval";			to: "cramersVCiLevel"	}
+		ChangeRename{	from: "hypothesis";									to: "alternative"		}
+		ChangeJS
+		{
+			name:		"alternative"
+			jsFunction:	function(options)
+			{
+				switch(options["alternative"])
+				{
+					case "groupsNotEqual":		return "twoSided";
+					case "groupOneGreater":		return "greater";
+					case "groupTwoGreater":		return "less";
+				}
+			}
+		}
+		ChangeRename{	from: "plotPosteriorOddsRatio";					to: "posteriorOddsRatioPlot"				}
+		ChangeRename{	from: "plotPosteriorOddsRatioAdditionalInfo";	to: "posteriorOddsRatioPlotAdditionalInfo"	}
```

Comment:
"posteriorOddsRatioPlotAddInfo" would be more consistent with the Bayesian correlation

Reply:
I'll change this in Bayesian correlations